AzureDay.WebApp/AzureDay.WebApp.Notification/Email/Template/ConfirmRegistration.cs

```diff
@@ -0,0 +1,332 @@
+﻿// ------------------------------------------------------------------------------
```

Comment:
the whole file is redundant and should be removed, registration will be handled by azure ad b2c